Mail/Components/AvatarView.swift

```diff
@@ -22,16 +22,34 @@ import MailResources
 import NukeUI
 import SwiftUI
 
-struct AvatarView: View, Equatable {
+struct AvatarView: View, Equatable, Identifiable {
```

Comment:
I understand that you marked `Identifiable` because the equality is tied to an id but this is kind of an anti-pattern as Identifiable is used for classes and structs tied to the view.